Sources/Commands/init.swift

```diff
@@ -30,10 +30,25 @@ extension InitError: CustomStringConvertible {
     }
 }
 
+/// Create an initial template package.
 final class InitPackage {
+    let rootd = POSIX.getcwd()
+
+    /// The mode in use.
     let mode: InitMode
+
+    /// The name of the example package to create.
     let pkgname: String
```

Comment:
Makes a lot of sense to minimize the diff.  We can always clean this up later.  It was indeed a nit-pick.